tabs-tabs-tabs/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "manifest_version": 2,
+  "name": "Tabs, tabs, tabs",
+  "version": "0.1",
+  "applications": {
+    "gecko": {
+      "id": "tabs-tabs-tabs@mozilla.org"
+    }
+  },
+  "browser_action": {
+      "default_title": "Tabs, tabs, tabs",
+      "default_popup": "tabs.html"
+  },
+  "permissions": [
+      "tabs"
```

Comment:
Trailing comma here stops this from installing. You don't need the "tabs" permission for the operations in this extension, so I think we should just remove this.